pages/index.tsx

```diff
@@ -30,16 +40,20 @@ const HomePage: HomePageComponent = () => {
 			<Head>
 				<title>{t('homepage.page-title')}</title>
 			</Head>
-			<HomeLayout>
-				<Container>
-					<Hero />
-					<Assets />
-					<ShortList />
-					<Earning />
-					<Features />
-					<TradeNow />
-					<GitHashID />
-				</Container>
+			<HomeLayout setCurrentPage={setCurrentPage}>
+				{currentPage === 'landing-page' ? (
+					<Container>
+						<Hero />
+						<Assets />
+						<ShortList />
+						<Earning />
+						<Features />
+						<TradeNow />
+						<GitHashID />
+					</Container>
+				) : (
+					<Stats />
```

Comment:
Not sure this is the best way to navigate for this page.

@platschi This routing means the stats dashboard is only accessible from the nav bar on the home page. Not ideal, but also we can't move into a main nav tab unless we have light mode designs.

Reply:
Yeah for the time being I guess it will live on the landing page as designed, next step should be light theme and adding it to main nav tab or dashboard sidebar.